visvis2/material/_base.py

```diff
@@ -1,2 +1,6 @@
 class Material:
-    pass
+    def __init__(self):
+        self.dirty = True
+        self.shaders = {}
+        self.uniforms = {}
+        self.primitiveTopology = None
```

Comment:
I can imagine that having a method to get these info can be beneficial in some cases, e.g. to to some JIT logic. But let's see, maybe this will hold up just fine :)

Reply:
this _is_ the info :)

Reply:
Haha, I get that. I meant *with a getter* to support the info to be more "dynamic" where needed. Anyway, we might never need it, so let's not overdesign now.